visualization/StackViz.js

```diff
@@ -425,8 +425,8 @@ export class StackViz {
             { fill: self._disabledColor, opacity: opacity, stroke: stroke },
           ];
         }
-        let high = self.r0ValueAtOffset(series, offset, "r0_h50");
-        let low = self.r0ValueAtOffset(series, offset, "r0_l50");
+        let high = self.r0ValueAtOffset(series, offset, "r0_h80");
+        let low = self.r0ValueAtOffset(series, offset, "r0_l80");
```

Comment:
This looks like an error in reading the value from the series. This is the issue I mentioned in the suggestion where the values were `NaN`. Just doing a spot check, this looks like the right data.